internal/cli/environment/create/k3d_defaults.go

```diff
@@ -9,7 +9,7 @@ var k3dDefaultConfigTmpl string
 
 const (
 	// k3dDefaultNodeImage defines default Kubernetes image for a new k3d cluster.
-	k3dDefaultNodeImage = "docker.io/rancher/k3s:v1.20.7-k3s1"
+	k3dDefaultNodeImage = "docker.io/rancher/k3s:v1.20.10-k3s1"
```

Comment:
Can you update it as well for kind? So, we use the same version in both cases? Kind defaults to 1.19.

Perfectly, we could have a base version in a one variable.

Reply:
I updated kind to 1.20.7 because they don't have 1.20.10, so I didn't introduce one variable for it.